source/documentation/other-topics/custom-domain-cert.md

```diff
@@ -7,28 +7,35 @@ cluster, this would be `*.apps.live-1.cloud-platform.service.justice.gov.uk`. As
 long as it is defined on the `Ingress` resource, it works automatically with a
 wildcard TLS certificate.
 
-However, most applications will typically need to be served on a `gov.uk`
-hostname. These hostnames (or usually, DNS zones) are managed externally,
-relative to the cluster, and there is a number of actions in order to set them
+However, most applications will typically need to be served on their own,
+application-specific `gov.uk` hostname. These hostnames (or usually, DNS zones)
+are managed individually and there is a number of actions in order to set them
 up for usage.
 
 #### Setup
 
-##### You do not have a DNS zone for the desired hostname
-Please [create a support ticket](http://goo.gl/msfGiS) providing as much
-information as possible.
+##### Defining the DNS zone
 
-Once the zone is ready, you can continue with the rest of this guide.
+To create the zone, you simply need to define it as a resource in your
+environment. Make sure to read [the guidance on naming domains][naming-domains]
+first and follow the instructions to [create the Route53 zone][creating-zone].
+To simplify management, we recommend that you only define a single zone, as part
+of the resources for your production environment, if possible.
 
-##### You already have a DNS zone for the desired hostname
+Once the zone is created, you will need to setup the necessary NS records in the
```

Comment:
Can we say "nameserver records" with a link to https://en.wikipedia.org/wiki/Name_server#Domain_Name_Server or some other page explaining what they are?